README.md

```diff
@@ -6,21 +6,44 @@ Currently supports classic & bendy butt feed types.
 
 ## API
 
-### `directMessageKey(x_dh_secret, x_dh_public, x_feed, y_dh_public, y_feed) => dmKey`
+### `directMessageKey(x_dh_secret, x_dh_public, x_feed_id, y_dh_public, y_feed_id) => dmKey`
+
+If you are _encrypting_ a DM, `x` is if your feed, and `y` is the person you are DM'ing.
+If you are _decryping_ a DM, `y` is if your feed, and `x` is the message author's.
```

Comment:
this should read

> If you are _decryping_ a DM, `x` is if your feed, and `y` is the message author's.

Reply:
you mean "is if your feed" => "is your feed", right?

Reply:
haha, yes that too!